Testscripts/Windows/SRIOV-RUN-COMMON-TEST.ps1

```diff
@@ -138,6 +140,19 @@ function Main {
         if ($CurrentTestData.Timeout) {
             $timeout = $CurrentTestData.Timeout
         }
+
+        # Determine if NIC_COUNT VFs are present in dependency VM before running remote test script.
+        Write-LogInfo "Checking VF count in dependency VM."
+        $cmdToSend = "find /sys/devices -name net -a -ipath '*vmbus*' | grep pci | wc -l"
```

Comment:
SC2126: Consider using grep -c instead of grep|wc -l.

Reply:
done